applications/app/controllers/InteractiveController.scala

```diff
@@ -64,10 +64,14 @@ class InteractiveController(
     s"$cdnPath/service-workers/$stage/$deployPath/$file"
   }
 
-  private def lookup(
-      path: String,
-  )(implicit request: RequestHeader): Future[Either[(InteractivePage, Blocks), Result]] = {
-    val result = capiLookup.lookup(path, range = Some(ArticleBlocks)) map { response =>
+  private def lookupItemResponse(path: String)(implicit request: RequestHeader): Future[ItemResponse] = {
+    capiLookup.lookup(path, range = Some(ArticleBlocks))
+  }
+
+  def fItemResponseToFE(fir: Future[ItemResponse])(implicit
```

Comment:
`fir` is a bit confusing. Perhaps `item`? I'd generally discourage encoding the type (Future) in the name here. Even better, make the method operate on `ItemResponse` and have the caller(s) handle the Future part. Especially as you are later doing e.g. `fItemResponseToFE(Future.successful(ir))`.